tailwind.config.ts

```diff
@@ -1,2 +1,20 @@
-// this file is just here for intellisense
-// see: https://github.com/tw-in-js/vscode-twind-intellisense/issues/24#issuecomment-1670633807
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./app/**/*.{js,ts,jsx,tsx,mdx}",
+  ],
```

Comment:
not in the tailwind docs, but if this uses standard bracket expansion, this should be like this (needs to be tested):
```suggestion
  content: [
    "./{app,components,pages}/**/*.{js,ts,jsx,tsx,mdx}",
  ],
```

if using `src` only, we can just use `"./src/**/*.{js,ts,jsx,tsx,mdx}"`
see: https://tailwindcss.com/docs/guides/nextjs

Reply:
If you want to make an issue for this or we can discuss later if we want to move to src directory. For now, the default is fine.